lms/djangoapps/courseware/views/index.py

```diff
@@ -179,11 +179,11 @@ def _init_new_relic(self):
 
     def _verify_position(self):
         """
-        Verify that the given position is in fact an int.
+        Verify that the given position is in fact an int and at least 1.
```

Comment:
Actually, we should update the name of the function to reflect this, not just the docstring.  Make sure it isn't used elsewhere, and then update to something like `_clean_position`.